### PR TITLE
Closes #3526, Closes #3534, and Closes #3538 - Fixes links, typos, style, and paragraph reflows, 

### LINKED ISSUE #3526: Typo
Hello,

Looks like you have a typo in the phrase below, the word "override" is missing one "r":

 **Overide** the ProcessRecord method to process
    // the supplied user name and write out a
    // greeting to the user by calling the WriteObject
    // method.
    protected override void ProcessRecord()

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ba54a8a2-201d-e028-8d96-75494779fef3
* Version Independent ID: aedf2455-91de-433f-4680-a1da3cb0088b
* Content: [How to Write a Simple Cmdlet](https://docs.microsoft.com/en-us/powershell/developer/cmdlet/how-to-write-a-simple-cmdlet)
* Content Source: [developer/cmdlet/how-to-write-a-simple-cmdlet.md](https://github.com/PowerShell/PowerShell-Docs/blob/staging/developer/cmdlet/how-to-write-a-simple-cmdlet.md)
* Product: **powershell**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### LINKED ISSUE #3534: Microsoft.Powershell.Scheduledjob link broken
On this page ("Scheduling Jobs with the Windows PowerShell API"), there's a link to a `Microsoft.Powershell.Scheduledjob` namespace, but the link is broken.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bbbe17d4-acdb-ff44-f467-e69d6eecb25a
* Version Independent ID: 522593d2-0871-0b06-0884-57905a419775
* Content: [Scheduling Jobs with the Windows PowerShell API](https://docs.microsoft.com/en-us/powershell/developer/workflow/scheduling-jobs-with-the-windows-powershell-api)
* Content Source: [developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/PowerShell/PowerShell-Docs/blob/staging/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md)
* Product: **powershell**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### LINKED ISSUE #3538: Add link to Get-WinEvent
Under "Related Links" please add a link to Get-WinEvent, since it is mentioned in the "Notes" section, but not linked there either.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf280305-63b5-a41b-fe39-4c50484029c6
* Version Independent ID: 8fd10e62-dee2-56fd-e3c1-07c07f8854a3
* Content: [Get-EventLog (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1#related-links)
* Content Source: [reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md)
* Product: **powershell**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in versions 3.0 - 5.1of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

Acrolinx scores
how-to-write-a-simple-cmdlet.md
Original version: 85
Updated version: 96

Get-EventLog.md
Original version: 84
Updated version: 92

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89e5d86](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/89e5d86846c755bcb06fb721482a28ee5e1c76fc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89e5d86](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/89e5d86846c755bcb06fb721482a28ee5e1c76fc)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 19 sec(s)_

File: developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md [hunk 1]
```diff
@@ -9,95 +9,107 @@ ms.topic: "article"
...
 schedJobDefinition.Register();
-                registrationSucceeded = true;
-                Console.WriteLine("Scheduled job has been registered.  Waiting 30 seconds for it to be started and run.");
+    registrationSucceeded = true;
+    Console.WriteLine("Scheduled job is registered. Waiting 30 seconds for it to run.");
 
 ```
 
-## Complete Code Example
+## Complete code Example
```
Comment (davidsmatlak) [inline developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md]: To stay within the line character limits, I inserted some line breaks in the code sample.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89e5d86](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/89e5d86846c755bcb06fb721482a28ee5e1c76fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/cmdlet/how-to-write-a-simple-cmdlet.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/cmdlet/how-to-write-a-simple-cmdlet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/how-to-write-a-simple-cmdlet?branch=pr-en-us-3576) |
[developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/workflow/scheduling-jobs-with-the-windows-powershell-api?branch=pr-en-us-3576) |
[reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-3.0&branch=pr-en-us-3576) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-4.0&branch=pr-en-us-3576) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.0&branch=pr-en-us-3576) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1&branch=pr-en-us-3576) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901181846539912-3576%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fSUBH3JWM0oMaRi5Ei8lfPp%2B3M7TRcKhy4rmbnNG7OA%3D&st=2019-01-18T18%3A55%3A55Z&se=2019-02-18T19%3A00%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b28e40d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/b28e40d2c632fc566f9efafdf8979aead8328e2e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

File: developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md (hunk 1 above)
Comment (sdwheeler) [reply developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md]: You didn't need to break those lines. I fixed it.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b28e40d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/b28e40d2c632fc566f9efafdf8979aead8328e2e)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 19 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 2]
```diff
@@ -9,155 +9,299 @@ title:  Get-EventLog
...
-Get-EventLog [-ComputerName <String[]>] [-List] [-AsString] [<CommonParameters>]
+Get-EventLog [-ComputerName <string[]>] [-List] [-AsString] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Get-EventLog cmdlet gets events and event logs on the local and remote computers.
 
-Use the parameters of Get-EventLog to search for events by using their property values.
-Get-EventLog gets only the events that match all of the specified property values.
+The **Get-EventLog** cmdlet gets events and event logs on the local and remote computers.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md]: \*\*Get-EventLog\*\*  should be \`Get-EventLog\`.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b28e40d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/b28e40d2c632fc566f9efafdf8979aead8328e2e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/cmdlet/how-to-write-a-simple-cmdlet.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/cmdlet/how-to-write-a-simple-cmdlet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/how-to-write-a-simple-cmdlet?branch=pr-en-us-3576) |
[developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/workflow/scheduling-jobs-with-the-windows-powershell-api?branch=pr-en-us-3576) |
[reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-3.0&branch=pr-en-us-3576) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-4.0&branch=pr-en-us-3576) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.0&branch=pr-en-us-3576) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1&branch=pr-en-us-3576) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901182143063467-3576%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=E0aEivpIvtyVgxG8%2BYY%2Bp62Np%2FGe7%2BN%2FVuQV6jNk6ng%3D&st=2019-01-18T21%3A51%3A31Z&se=2019-02-18T21%3A56%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Changed in all versions.

File: developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md (hunk 1 above)
Comment (davidsmatlak) [reply developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md]: Thank you Sean!

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9bed6bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/9bed6bf099f5a37385ee8e6878612de5b19dda8f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9bed6bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/9bed6bf099f5a37385ee8e6878612de5b19dda8f)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 17 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9bed6bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3576/commits/9bed6bf099f5a37385ee8e6878612de5b19dda8f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/cmdlet/how-to-write-a-simple-cmdlet.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/cmdlet/how-to-write-a-simple-cmdlet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/how-to-write-a-simple-cmdlet?branch=pr-en-us-3576) |
[developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/workflow/scheduling-jobs-with-the-windows-powershell-api?branch=pr-en-us-3576) |
[reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-3.0&branch=pr-en-us-3576) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-4.0&branch=pr-en-us-3576) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.0&branch=pr-en-us-3576) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue-3526/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1&branch=pr-en-us-3576) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901182233126941-3576%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1%2BTNG7RYDREVYi%2BAolRjklPpmS2VyfNSIuCHN7VUeyY%3D&st=2019-01-18T22%3A42%3A10Z&se=2019-02-18T22%3A47%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**